lib/matplotlib/table.py

```diff
@@ -178,23 +196,6 @@ def set_text_props(self, **kwargs):
         self._text.update(kwargs)
         self.stale = True
 
-
-class CustomCell(Cell):
```

Comment:
As this name is public API, it still needs to go through a deprecation period before being removed. You can use the helper decorators in `matplotlib.cbook` to implement it.